### PR TITLE
add python3-transitions rosdep key

### PR DESCRIPTION
**Pull request by zhtroy**
Please add the following dependency to the rosdep database.

## Package name:

python3-transitions

## Package Upstream Source:

https://github.com/pytransitions/transitions

## Purpose of using this:

after(including) ubuntu:bionic， python transitions is updated to python3 version, I'm using it in my project

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/search?keywords=python3-transitions&searchon=names&suite=all&section=all
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=python3-transitions&searchon=names

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5256,6 +5256,16 @@ python-transitions:
...
+python3-transitions:
+  debian: 
+    '*': [python3-transitions]
+    stretch:
+      pip:
+        packages: [transitions]
+  ubuntu:
+    '*': [python3-transitions]
+    xenial: null
+    trusty: null
```
Comment (ivanpauno) [inline rosdep/python.yaml:5268]: you could add the pip key for this two as well

**Comment by ivanpauno** [comment]
@zhtroy could you check the CI failures?

> rosdep/python.yaml#L5260
> Package 'python3-transitions' could not be found for debian buster on amd64

> rosdep/python.yaml#L5260
> [trailing-spaces] trailing spaces

File: rosdep/python.yaml (hunk 1 above)
Comment (zhtroy) [reply rosdep/python.yaml]: added in new commit

**Comment by zhtroy** [comment]
fixed CI problem

**Review by ivanpauno — APPROVED** [review]

**Comment by zhtroy** [comment]
> @zhtroy mind to check for package availability in Fedora, Gentoo, and RHEL? Thanks!

sorry, don't have much time

**Comment by tfoote** [comment]
The CI fix has moved the `python-trep` incorrectly infront of `python-transitions`

**Review by tfoote — APPROVED** [review]

**Review by cottsay — APPROVED** [review]

**Pull request merged**